src/components/auth/Status.js

```diff
@@ -0,0 +1,13 @@
+// import React, { useState, useContext, useEffect } from "react";
+
+// export default () => {
+//     const [isLoggedIn, setIsLoggedIn] = useState(false);
+//     const { getSession } = useContext(AccountContext);
+
+//     useEffect(() => {
+//         getSession().then((session) => {
+//             console.log("Session: ", session);
+//             setIsLoggedIn(true);
+//         });
+//     }, [getSession]);
+// };
```

Comment:
Why is all of this commented out? If we aren't using it, let's delete.

Reply:
Was trying to make this sorta an util function lol. forgot to delete